jetbrains-core/src/software/aws/toolkits/jetbrains/utils/ZIpUtils.kt

```diff
@@ -0,0 +1,11 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.utils
```

Comment:
1) `ZIpUtils.kt` - the i should be lower-case
2) Don't we already have zip utils: `software.aws.toolkits.core.utils.ZipUtils`

Reply:
Fixed